_activities/cwp.md

```diff
@@ -29,7 +29,7 @@ we are reaching out to other HEP experiments around the world to participate.
 ## Final Roadmap Paper
 
 The second draft of the CWP, the 
-[Roadmap for HEP Software and Computing R&D for the 2020s](https://docs.google.com/document/d/1mXJ51Xi1FEcQKzf6jJknmkC3MGeZqjPjgtoj3pVD6rk){:target="_cwp_roadmap"} 
+[Roadmap for HEP Software and Computing R&D for the 2020s](https://docs.google.com/document/d/1RIcnj7DBNOoQ1DT45WCGFboS0tCDeJcvAi2xjPv_UVQ){:target="_cwp_roadmap"} 
 report, has been released to the community for comment (also available as a 
 [pdf]({{site.docs.url}}/CWP/papers/roadmap/HSF-Community-White-Paper-v0.2.pdf)). You are **encouraged to read it** and
 **provide feedback**. The document will remain open for comments **until December 1**.
```

Comment:
This text is out of date. Can we update it to "Comments were received until 4 December and the final paper is now being prepared."

Reply:
Also, remove the next paragraph that describes the mechanics of commenting.